app/src/apdu_handler.c

```diff
@@ -44,7 +44,7 @@ __Z_INLINE void handleGetAddr(volatile uint32_t *flags, volatile uint32_t *tx, u
 
     if (requireConfirmation) {
         view_review_init(addr_getItem, addr_getNumItems, app_reply_address);
-        view_review_show();
+        view_review_show(0x03);
```

Comment:
Please, replace the number with the value from enum **review_type_e** from view_internal.h

Reply:
yeh, makes a lot of sense 👍🏻